README.md

```diff
@@ -1,27 +1,24 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/helmclient.svg?&style=shield)](https://circleci.com/gh/giantswarm/helmclient)
+[![GoDoc](https://godoc.org/github.com/giantswarm/helmclient?status.svg)](http://godoc.org/github.com/giantswarm/helmclient) [![CircleCI](https://circleci.com/gh/giantswarm/helmclient.svg?&style=shield)](https://circleci.com/gh/giantswarm/helmclient)
 
 # helmclient
 
-Package helmclient implements Helm related primitives to work against Tiller.
+Package helmclient implements [Helm] related primitives to work against helm
+releases. Currently supports Helm 2 and connects to the Tiller gRPC API using
+a port forwarding connection.
 
-## Projects using `helmclient`
```

Comment:
This projects using is out of date and a pain to keep updated. I think it's best to just drop it.